src/classes/BDI_ObjectMapping.cls

```diff
@@ -84,10 +84,13 @@ public virtual with sharing class BDI_ObjectMapping {
         this.Custom_Mapping_Logic_Class = objectMapping.Custom_Mapping_Logic_Class__c;
         this.Data_Import_Object_Mapping_Set = objectMapping.Data_Import_Object_Mapping_Set__c;
         this.Data_Import_Object_Mapping_Set_Dev_Name = objectMapping.Data_Import_Object_Mapping_Set__r.DeveloperName;
-        this.Imported_Record_Field_Name = objectMapping.Imported_Record_Field_Name__c;
-        this.Imported_Record_Status_Field_Name = objectMapping.Imported_Record_Status_Field_Name__c;
-        this.Object_API_Name = objectMapping.Object_API_Name__c;
+        this.Imported_Record_Field_Name = UTIL_Describe.getFieldName(SObjectType.DataImport__c.Name, 
+                                                                    objectMapping.Imported_Record_Field_Name__c);
+        this.Imported_Record_Status_Field_Name = UTIL_Describe.getFieldName(SObjectType.DataImport__c.Name, 
```

Comment:
Style update only, but can we avoid the long line and add a linebreak in here?

Reply:
I don't think we can do a line break here. It doesn't look excessively long in VSCode.